packages/curve-ui-kit/.storybook/preview.tsx

```diff
@@ -21,10 +22,20 @@ export const decorators: Decorator[] = [
     Provider: ThemeProvider,
     GlobalStyles: CssBaseline,
   }),
-  (Story) => (
-    <>
-      <style>
-        {`
+  (Story) => {
+    const router = createRouter({
+      routeTree: createRootRoute({
+        component: () => <Story />,
```

Comment:
```suggestion
        component: Story,
```

Reply:
Will pick that up [in this PR](https://github.com/curvefi/curve-frontend/pull/1321)